### DISCUSSION TITLE
Hexo评论系统对比推荐 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#2] [Announcements]
# Hexo评论系统对比推荐 - 没有秃头基因的燊的博客

突然期末考的时间出了，搞得火急火燎地看了两天，然后突然又说给带ppt进考场，感觉可以放松一下就写篇之前一直想写的博客吧。本篇包含Giscus, Utterances, Gitalk, Gitment, Waline, Twikoo, Valine。

https://yzs020220.github.io/2023/02/01/hexo%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F%E5%AF%B9%E6%AF%94/

**Comment by wangyuyao438** [comment]
hello

**Comment by Allinaent** [comment]
我用的gitlab的page，不能用giscus怎么办呢？

**Reply by yzs020220** [reply]
gitlab貌似也可以将hexo托管上去，具体的我没有仔细了解

**Reply by Allinaent** [reply]
gitlab没有disscussion，我用的是hugo stack，评论这种东西，果然还是github好用。后面有时间迁到github，谢谢回复。还看了waline，这个需要vercel，域名加cname做个dns映射。没时间弄。算了，没有评论也挺好的。/(ㄒoㄒ)/~~

**Comment by AdingApkgg** [comment]
`Twikoo`、`Waline`、`Valine`、`Giscus` 一起用还挺爽的..

**Comment by DominicLiang** [comment]
这个评论系统是Giscus吗?

**Reply by yzs020220** [reply]
是的

**Comment by yucho123987** [comment]
我用的是Waline（没错，我有自己的域名），最近在自己写一个hexo主题，目前在尝试尽可能多地兼容各种评论系统[doge]

**Reply by DominicLiang** [reply]
> 我用的是Waline（没错，我有自己的域名），最近在自己写一个hexo主题，目前在尝试尽可能多地兼容各种评论系统[doge]

你好. 我也想自己写一个hexo主题, 但是hexo我不太熟. 方便加个qq交流一下吗?

**Reply by yucho123987** [reply]
> > 我用的是Waline（没错，我有自己的域名），最近在自己写一个hexo主题，目前在尝试尽可能多地兼容各种评论系统[doge]
> 
> 你好. 我也想自己写一个hexo主题, 但是hexo我不太熟. 方便加个qq交流一下吗?

我的QQ：***

**Comment by bg51717** [comment]
Giscus是不是当一个人在一个页面下进行评论的时候，之前所有评论的人都会收到邮件通知？

**Reply by yzs020220** [reply]
是的，但好像可以进行设置，具体可以去搜索下

**Comment by Merinthae** [comment]
这个评论区也蛮特别的，要看完所有评论才可以评价😄

**Reply by Merinthae** [reply]
还可以对别人的评论进行评价诶！

**Comment by Minus2xy** [comment]
用gibhub评论对非魔法用户不太友好 ，我自己用了一下那个来必力，但是好像不是很稳定

**Reply by yucho123987** [reply]
要稳定还得自建
